### ISSUE TITLE
ui5-select accessible name via aria-label or aria-labelledby does not work

### ISSUE THREAD

**Issue by mastanshaikk** [SAP/ui5-webcomponents#2232]
**Describe the bug**
ui5-select accessible name via aria-label or aria-labelledby does not work because the aria-labelledby was set to use an internal label from lines: https://github.com/SAP/ui5-webcomponents/blob/master/packages/main/src/Select.hbs#L9 and https://github.com/SAP/ui5-webcomponents/blob/master/packages/main/src/Select.hbs#L22 where the external aria-labelledby attribute will not be applied that was provide on ui5-select element and aria-label attribute seem to apply, but aria-labelledby has a high precedence over aria-label so it always uses the internal label irrespective of aria-label or aria-labelledby attributes provided, a good solution could be to append the aria-labelledby id's or calculate effect accessible name and apply it. This is a problem if it is used in associated with a label like in a form.
![image](https://user-images.githubusercontent.com/48530083/93836312-5ad96000-fc50-11ea-9f11-b84d309e541c.png)

**To reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/Label/ try navigate to the ui5-select element though the label "Job" was associated it will not be read by screen readers.

**Expected behavior**
Should be able to associate an aria-label or aria-labelledby via attributes on ui5-select and have it reflected along with the selected option text.

**Screenshots**
![image](https://user-images.githubusercontent.com/48530083/93836312-5ad96000-fc50-11ea-9f11-b84d309e541c.png)

**Comment by fifoosid** [comment]
Hi @mastanshaikk 

The reported issue has already been reported(see #2142) and is currently being worked on. I will close the current issue as duplicate as the progress could be tracked in the linked issue

**Comment by mastanshaikk** [comment]
Hi @fifoosid 

Thanks, will follow #2142
